tests/e2e/unified_test.go

```diff
@@ -378,6 +378,7 @@ func runScenario(ctx context.Context, t *testing.T, testPause bool, fixture reso
 				}
 
 				if ShouldTestRereconiliation(t, testName, primaryResource) {
+					h.Log("Testing re-reconciliation...")
```

Comment:
nit: can we include the name of the primary resource or some other identifying string ?

Reply:
Done.